packages/x-components/src/components/layouts/fixed-header-and-asides-layout.vue

```diff
@@ -108,8 +109,8 @@
   })
   export default class FixedHeaderAndAsidesLayout extends mixins(LayoutsMixin) {
     protected scrollPosition = 0;
-    protected rightAsideAnimation = animateTranslate('right');
-    protected leftAsideAnimation = animateTranslate('left');
+    protected rightAsideAnimation: VueConstructor = animateTranslate('right');
+    protected leftAsideAnimation: VueConstructor = animateTranslate('left');
```

Comment:
I don't know why, if we let these two properties type to be inferred, it adds a `MainScroll` type there, which is incorrect and very strange.